tests/test_stat_collector.py

```diff
@@ -64,24 +64,34 @@ def test_simple_search_in_kapiche_ram(self, ram_kapiche_index):
         assert len(result.hits) == 5
         _, doc_address = result.hits[0]
 
-    def test_stat_searcher(self, ram_kapiche_index):
+    def test_stat_searcher_none(self, ram_kapiche_index):
         index = ram_kapiche_index
         query = index.parse_query("sea", ["body"])
-
-        result = index.stat_searcher().search(query, set())
```

Comment:
# NOTE: NOW AN EMPTY SET WILL RETURN NOTHING BECAUSE NOTHING WILL MATCH

Instead, to express "no filter", it either has to be missing or `None`.

When updating chrysalis, please take this into account.